library/src/scala/tasty/Reflection.scala

```diff
@@ -1348,9 +1348,22 @@ trait Reflection extends reflect.Types { reflectSelf: CompilerInterface =>
 
   object Type:
 
+    /** Get the Type representing the given class */
     def apply(clazz: Class[_])(using ctx: Context): Type =
       reflectSelf.Type_apply(clazz)
 
+    /** Get package type if package is either defined in current compilation run or present on classpath. */
```

Comment:
The documentation of these methods should describe what happens when the type is not found.

Reply:
Added